clouddriver-kubernetes/src/test/resources/com/netflix/spinnaker/clouddriver/kubernetes/op/job/mock-kubectl-stdin-command.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+#
+# Copyright 2022 Salesforce, Inc.
```

Comment:
2023? 😄 

LGTM though!

Reply:
Yeah, we've been using this for awhile...